psybot/modules/challenge.py

```diff
@@ -170,8 +206,27 @@ async def done(interaction: discord.Interaction, contributors: Optional[str]):
 
     await move_channel(interaction.channel, get_complete_category(interaction.guild))
 
-    msg = ":tada: {} was solved by ".format(interaction.channel.mention) + " ".join(f"<@!{user}>" for user in users) + " !"
-    await interaction.guild.get_channel(ctf_db.channel_id).send(msg)
+    # Special emojis for certain users, otherwise default
+    emojis = {
+        286173785336446978: (":fire:", "firepog"),  # firehawk
+        112158630161158144: (":punch:", "noshell"),  # N1z0ku
+        145301884662448128: (":flag_no:", "chadtor"),  # Victor4X
+        634877910511255575: (":lemon: :last_quarter_moon_with_face:", "lightsaberpepe"),  # Anakin
+    }
+    msg_emoji = ":tada:"
+    reaction_emoji = "peepoBrunner"
+
+    if len(users) == 1 and users[0] in emojis:
+        msg_emoji, reaction_emoji = emojis[users[0]]
+
+    solvers = " ".join(f"<@!{user}>" for user in users)
+    msg = f"{msg_emoji}  {interaction.channel.mention} was solved by {solvers}!  {msg_emoji}"
+    sent_msg = await interaction.guild.get_channel(ctf_db.channel_id).send(msg)
```

Comment:
The same in regards to character limit, can *technically* happen here, however *very* unlikely.

Reply:
#wontfix, that would require a list of over 80 solvers :kek:

Reply:
I think the bot is riddled with similar issues. So that's totally fine!

I just wanted to highlight it.